src/python/national_voter_file/us_states/ny/transformer.py

```diff
@@ -194,7 +202,7 @@ def extract_mailing_address(self, input_dict):
                 ]))
 
                 if address_type == 'Ambiguous':
-                    print("Warn - %s: Ambiguous mailing address falling back to residential (%s)" % address_type, input_dict['MAILADD1'])
+                    print("Warn - %s: Ambiguous mailing address falling back to residential (%s)" % (address_type, input_dict['MAILADD1']))
```

Comment:
Was this line used in other transformers? If so, it doesn't need to be in this PR, but we should fix it there too